src/ipfs/basic.ts

```diff
@@ -50,9 +51,10 @@ export const dagGet = async (cid: CID): Promise<DAGNode> => {
 
 export const dagPut = async (node: DAGNode): Promise<AddResult> => {
   const ipfs = await getIpfs()
-  // using this format so that we get v0 CIDs. ipfs gateway seems to have issues w/ v1 CIDs
+  // using this format because Gateway doesn't like `dag-cbor` nodes. 
+  // I think this is because UnixFS requires `dag-pb` & the gateway requires UnixFS for directory traversal
   const cidObj = await ipfs.dag.put(node, { format: 'dag-pb', hashAlg: 'sha2-256' })
-  const cid = cidObj.toString()
+  const cid = CIDTool.base32(cidObj.toString())
```

Comment:
Can we do `cidObj.toString('base32')`?
`js-ipfs` uses this package for cids right? https://www.npmjs.com/package/cids

Reply:
oh nice thanks for pointing that out! 
I actually had to do `cidObj.toV1().toString()` but I could remove the `cid-tool` dependency :ok_hand: